identity_credential/src/error.rs

```diff
@@ -45,6 +45,11 @@ pub enum Error {
   #[error("could not convert JWT to the VC data model: {0}")]
   InconsistentCredentialJwtClaims(&'static str),
 
+  /// Caused when deserializing a Presentation with an empty array for the
+  /// `verifiableCredential` property.
+  #[error("empty verifiableCredential array")]
+  EmptyVerifiableCredentialsArray,
```

Comment:
```suggestion
  #[error("empty verifiableCredential array in presentation")]
  EmptyVerifiableCredentialArray,
```

Nit: Would clarify where the error occured (in the presentation) and loose the `s` since the field doesn't have it :)

Reply:
I've adjusted the test to test for the specific error. I feel there must be a prettier way to do it but doing a string compare does the job as well.